### PR TITLE
Retriever is deactivated, see issue 3482

### PR DESCRIPTION
**Pull request by annando**
See issue https://github.com/friendica/friendica/issues/3482

Accidentally this PR includes some other coding that I made some time ago but had never done a PR for this.

This other PR simply is a whitelist for a blacklist ;-)

### CODE REVIEW

File: leistungsschutzrecht/leistungsschutzrecht.php [hunk 1]
```diff
@@ -71,9 +71,13 @@ function leistungsschutzrecht_fetchsites() {
 	$sitelist = fetch_url($url);
 	$siteurls = explode(',', $sitelist);
 
+	$whitelist = array('tagesschau.de', 'heute.de', 'wdr.de');
```
Comment (tobiasd) [inline leistungsschutzrecht/leistungsschutzrecht.php]: Wont it be better if the whitelist was configurable by the admin or user?

File: retriever/retriever.php [hunk 2]
```diff
@@ -4,6 +4,7 @@
  * Description: Follow the permalink of RSS/Atom feed items and replace the summary with the full content.
  * Version: 1.0
  * Author: Matthew Exon <http://mat.exon.name>
+ * Status: Unsupported
```
Comment (tobiasd) [inline retriever/retriever.php]: Could you add a readme file explaining what to do with core utils to get a similar functionality?

Comment (annando) [inline retriever/retriever.php]: I'm not totally sure what ```retriever``` is really doing. :-)

**Comment by annando** [comment]
I added some small README.

File: leistungsschutzrecht/leistungsschutzrecht.php (hunk 1 above)
Comment (annando) [inline leistungsschutzrecht/leistungsschutzrecht.php]: It would. Can I do this in a second PR?

Comment (tobiasd) [inline leistungsschutzrecht/leistungsschutzrecht.php]: I'm thinking about making it a blocking issue for the next release ;-)

**Comment by tobiasd** [comment]
Actually I think it's a non blocking issue, but would be really nice to have :-)

**Pull request merged**